doc/kuksa_analysis/kuksa_use_cases.md

```diff
@@ -0,0 +1,243 @@
+# KUKSA Use Cases
+
+
+# Content
+- [KUKSA Use Cases](#kuksa-use-cases)
+- [Content](#content)
+- [Use Cases status](#use-cases-status)
+- [Use cases:](#use-cases)
+  - [UC1](#uc1)
+  - [UC2](#uc2)
+  - [UC3](#uc3)
+  - [UC4](#uc4)
+  - [UC5](#uc5)
+  
+# Use Cases status
+| Description                               | Status                       |
+|-------------------------------------------|------------------------------|
+| Implemented and Verified                  | 🟢                           |
+| Approved, Not Yet Implemented             | 🟡                           |
+| Long Term Goal                            | 🔴                           |
+
+# Use cases:
+
+## UC1
+    Title: Signal consumber subscribes to all signal values.
+
+    Status: 🟡
+
+    Description: Signal Consumer subscribes on changed sensor values or actuator current values of all signals
+
+**Primary Actor:** Signal Consumer
+
+**Secondary Actors:** Databroker, Provider, Vehicle Network 
```

Comment:
The Databroker isn't an actor - it is passive and reacts to requests of consumers and providers. As all the use cases are related to it, imho we do not need to mention it here at all:
```suggestion
**Secondary Actors:** Provider, Vehicle Network 
```